src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -145,6 +147,23 @@ const Body = BaseFormWithPolling.extend(
       });
     },
 
+    showCustomFormErrorCallout(error) {
+      let options = {
+        type: 'error',
+        className: 'okta-verify-uv-callout-content',
+        subtitle: error.responseJSON.errorSummary,
+      };
+
+      const containsSignedNonceError = error.responseJSON.errorSummaryKeys
+        .some((key) => key.includes(SIGNED_NONCE_ERROR_KEY));
```

Comment:
can there be multiple keys in the response ?

Reply:
Yes, it is possible. That is why we are checking to have at least one match.
Also, in the case of multiple i18n codes. The message will be composed of localised i18n code messages, 
${loc(i18nKeys[0])} + '. ' + ${loc(i18nKeys[1])} + '. ' + ${loc(i18nKeys[n])}

this logic can be found [here](https://github.com/okta/okta-signin-widget/blob/f6b6246e537f0980acca98fa5d5421399fc30dbc/src/v2/ion/IonResponseHelper.js#L114)